dkc/core/models/folder.py

```diff
@@ -8,23 +8,21 @@
 from django.dispatch import receiver
 from django_extensions.db.models import TimeStampedModel
 
-from dkc.core.exceptions import MaxFolderDepthExceeded
 from dkc.core.models.metadata import UserMetadataField
 
+MAX_DEPTH = 30
```

Comment:
I had to make this module-scope. If it was an class attribute of `Folder`, it couldn't be referenced from the scope of `Folder.Meta`.